logs.rst

```diff
@@ -8,3 +8,14 @@ Results while adding implementations -
 |                                                           |  S2 - Shallow_800                |         5            |     0.8799          |
 |                                                           |  S3 - Shallow_800                |                      |     0.98            |
 +-----------------------------------------------------------+----------------------------------+----------------------+---------------------+
+
+Results for Pruning using Lottery Tickets - 
+===========================
+
++-----------------------------------------------------------+----------------------------------+----------------------+---------------------+
+|  Model used                                               |  Pruning percent                 | Epochs               | Test accuracy(ies)  |
++===========================================================+==================================+======================+=====================+
+|  ResNet18 (on MNIST)                                      |  0%                              |                      |     0.9829          |
+|                                                           |  10%                             |         3            |     0.9816          |
+|                                                           |  19%                             |                      |     0.9851          |
++-----------------------------------------------------------+----------------------------------+----------------------+---------------------+
```

Comment:
Can you make a column for dataset separately, and please do it for DML as well. Even that is trained on MNIST as far as I remember.

Also, sorry for not mentioning before, train it for 5 epochs to maintain uniformity. 

cc: @NeelayS